backend/supabase/users/create.sql

```diff
@@ -0,0 +1,12 @@
+create table if not exists
```

Comment:
I'm not sure it's better to have a separate file for each of the table creation, indexes, rls, and triggers.

Seems better to put those all in the same file. I guess the advantage here is you can see if, say, rls exists by looking at the filenames? Not sure that outweighs colocality.